Makefile

```diff
@@ -484,7 +484,7 @@ uninstall:
 # Linters --------------------------------------------- {{{1
 #
 
-PYTHON_SCRIPTS := utils/hare-shutdown utils/hare-status utils/gen-uuid utils/utils.py
+PYTHON_SCRIPTS := $(shell grep 'python3' -n utils/* 2>/dev/null | grep ':1:' | sed 's/^\([^:]*\):.*$/\1/g') utils/utils.py
```

Comment:
Why `utils/utils.py` is required again? the command output includes the file already,
```
[520478@ssc-vm-1623:cortx-hare-1] grep 'python3' -n utils/* 2>/dev/null | grep ':1:' | sed 's/^\([^:]*\):.*$/\1/g'
utils/gen-uuid
utils/hare-fetch-fids
utils/hare-shutdown
utils/hare-status
utils/utils.py
```

Reply:
Okay I'll remove this utils/utils.py.

Reply:
If I remove utils/utils.py then "make" command gives error like  
    
 ./hax/hax/motr/__init__.py:400:75: W504 line break after binary operator
./hax/hax/motr/__init__.py:409:71: W504 line break after binary operator
./hax/hax/motr/__init__.py:427:72: W504 line break after binary operator
./hax/hax/motr/delivery.py:59:72: W504 line break after binary operator
./hax/hax/motr/delivery.py:60:67: W504 line break after binary operator
make[1]: *** [flake8] Error 1
make: *** [build] Error 2                                     

So it is added in this list.

Reply:
Please note that `./hax/hax/motr/init.py` is not part of utils folder. So it is strange that those files start appearing in the output.

And moreover, it is strange that this error IS NOT captured by hax-related checks in Makefile. @supriyachavan4398 Please investigate.

Reply:
Done all changes.